### ISSUE TITLE
use es5 import statements

### ISSUE THREAD

**Issue by SwiftMike** [SwiftMike/hiigrade#39]

**Comment by SwiftMike** [comment]
Replace all "import" statements with "require" for deployment issues

**Comment by yoonslee** [comment]
This is just server-side code, correct? So, anything in the `client` folder, we don't have to worry about?

If we can set a linting rule for server code only, that would be awesome to make sure we are thorough.

I can help out with this if you'd like, @SwiftMike .

**Comment by yoonslee** [comment]
Pushed up some changes to `plant-group-yoon` that will make this task easier.

In command-line, run `npm run lint:server` and it will show any non-ES5 code as lint errors.

@SwiftMike Gonna start going through it, has like 182 errors

**Comment by yoonslee** [comment]
Pushed up some changes to `plant-group-yoon2` that will make this task easier.

Added a .eslintrc file that only applies to the server folder.

**Comment by SwiftMike** [comment]
Yep! I'm guessing we only have to use this for the server side code, but I thought the .babelrc file would handle all of the es6 requests but I found out how to fix the errors. They will be closed by tonight.
